cartography/intel/aws/ssm.py

```diff
@@ -0,0 +1,192 @@
+import logging
+from typing import Any
+from typing import Dict
+from typing import List
+
+import boto3
+import neo4j
+
+from cartography.util import aws_handle_regions
+from cartography.util import dict_date_to_epoch
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+def get_running_instance_ids(neo4j_session: neo4j.Session, region: str, current_aws_account_id: str) -> List[str]:
```

Comment:
This means that the instances must exist in the graph before we can begin discovering SSM patches and information. I think it would be better sort of do the discovery from both directions:
1. query AWS for SSM Information and Patches
2. Load those into the graph as SSMInstanceInformation and SSMInstancePatch nodes
  1. Also use the instance_id to create what I like to call husk EC2Instance nodes, where they only have the id property. 
 
This is how we do it for EC2Instance and EC2Subnet.
https://github.com/lyft/cartography/blob/master/cartography/intel/aws/ec2/instances.py#L191
https://github.com/lyft/cartography/blob/master/cartography/intel/aws/ec2/subnets.py

Reply:
I understand. That makes me want to add this directly to the ec2 module, just because they are so dependent. But I'm okay with `ssm` being by itself.